azurerm/resource_arm_function_app.go

```diff
@@ -799,10 +820,68 @@ func flattenFunctionAppIdentity(identity *web.ManagedServiceIdentity) interface{
 	if identity.TenantID != nil {
 		result["tenant_id"] = *identity.TenantID
 	}
+	if identity.UserAssignedIdentities != nil {
+		result["user_assigned_identity"] = flattenFunctionAppUserAssignedIdentities(identity.UserAssignedIdentities)
+	}
```

Comment:
can we remove this `if` statement and return an empty slice/array from the `flattenFunctionAppUserAssignedIdentities` function - to ensure this is set to an empty list if Azure returns a bad response here?